### ISSUE TITLE
Extract doesn't work with new i18n._ signature?

### ISSUE THREAD

**Issue by snegostup** [lingui/js-lingui#181]
**tl;dr** when using new `i18n._` signature, output works but extraction doesn't. When using old signature - extraction works, but output doesn't.

According to [new docs](https://lingui.github.io/js-lingui/releases/migration-2.html#signature-of-i18n-changed) it should be used like this
```javascript
i18n._('a.id',
   {val: "John"},
   {defaults: "My name is {val}"});
```
But in this case `extract` doesn't pick up the `defaults`

_messages.json:_
```json
"a.id": {
    "translation": "",
    "origin": [
      [
        "src/components/file.js",
        64
      ]
    ]
  },
```
When used with old signature
```javascript
i18n._('b.id', {
   values: {val: "Liza"},
   defaults: "My name is {val}",
});
```
it extracts properly:
```json
"b.id": {
    "translation": "",
    "defaults": "My name is {val}",
    "origin": [
      [
        "src/components/file.js",
        69
      ]
    ]
  }
```
but doesn't use the variable in output:
```javascript
const b = i18n._('b.id', {
   values: {val: "Liza"},
   defaults: "My name is {val}",
});
console.log("b: ", b);
// outputs: b:  My name is <-- Liza is missing
```

Versions:
"@lingui/react": "^2.0.5",

CLI:
$ lingui --version
2.0.5

**Comment by snegostup** [comment]
After looking at `@lingui/babel-plugin-extract-messages` I can confirm that when processing `i18n._` it doesn't look at third argument (which sets the `defaults`) and looks for `defaults` in second argument as in older signature.

@tricoder42 can you please suggest how can I use `i18n` outside react -
1) Using custom id
2) With properly working extraction
3) With properly working output

Thanks

**Comment by snegostup** [comment]
A little temporary workaround I've come up with:
```javascript
import {setupI18n} from '@lingui/core';

const realI18n = setupI18n({
  language: langCode,
  catalogs: {[langCode]: langCatalog}
});

// fake i18n._ with old signature for extract, returns args for new signature
let i18n = {
  _: (id, options) => [
    id,
    options.values,
    {defaults: options.defaults}
  ]
};

// using fake i18n._ with old signature
const args = i18n._('message.id', {
  values: {val},
  defaults: "The value is {val}"
});

const res = realI18n._(...args);
```

**Comment by tricoder42** [comment]
Fixed and released in **2.0.7**. 🎉 

Thank you for initial PR!
